### ISSUE TITLE
Inventory t-shirt stock

### ISSUE THREAD

**Issue by jhibbets** [ashumz/citycamp#384]
Can someone do a quick inventory or our t-shirt stash? how many, what years, what sizes?

No rush on this. Rainy day / hot day material here.

**Comment by ChrisTheDBA** [comment]
this is easy.  Zero, none, nada...every one gone.  As Hemingway wrote "Nada y nada, y pues nada"

**Comment by jhibbets** [comment]
I think we still had some leftover in a purple and green container. (or is
that in my car?)

**Comment by ChrisTheDBA** [comment]
true.  Don't know know what's in your car. :)

**Comment by jhibbets** [comment]
So you and Tracy don't have the extra shirts? I'll do this task then, sorry
about that.

**Comment by ChrisTheDBA** [comment]
Nope, she said everything was gone.  the GDI students liked them!

**Comment by ChrisTheDBA** [comment]
Patrick grabbed a bunch of stuff too.
